Documentation/markdown/BuildOptions.md

```diff
@@ -0,0 +1 @@
+Here will be BuildOptions.md file 
```

Comment:
Remove this file, @ccoetzee got this as task. It is named CMakeOptions.md for your reference.

Reply:
Done